mlir/test/Dialect/MHAL/emulate-narrow-type.mlir

```diff
@@ -0,0 +1,31 @@
+// RUN: rocmlir-opt -mhal-emulate-narrow-type -canonicalize \
```

Comment:
This change is in wrong commit. Please move it to the MHAL commit

Reply:
It's not in an MHAL commit because it's not being applied to the MHAL directory

Reply:
yep thats the problem... the pass definition belongs to /external/mhal ?
Why do we have the test in rocMLIR